rel/ballcolor.cpp

```diff
@@ -14,22 +14,28 @@
 
 namespace ballcolor {
 
-static u8 convert_to_ballColorID(u8 color_choice){
-    if(color_choice == 0) return 3;
-    else return color_choice - 1;
+static u8 s_convert_to_ball_color_id(u8 color_choice) {
+    if (color_choice == 0)
+        return 3;
```

Comment:
I prefer multiline `if`s to use curly braces, that way you can never accidentally do:

```c
if (blah)
    thing1();
    thing2(); // Oops, always runs!
```

Reply:
This was originally 1-line, but the auto-formatter split it into two lines. For future reference, should I just always use curly braces & multiple lines, so the auto formatter doesnt make this happen?